packages/components/src/components/hds/form/radio-card/index.hbs

```diff
@@ -3,7 +3,7 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<label class={{this.classNames}} {{did-insert this.setAriaDescribedBy}} {{style maxWidth=@maxWidth}}>
```

Comment:
The descriptors are never updated dynamically and never have been. the did-insert here is un-necessary since the introduction of this component: https://github.com/hashicorp/design-system/commit/5edb999bae52a5f1bf1dad68a983e57d7993e3b5